parser_test.go

```diff
@@ -1874,11 +1874,11 @@ func TestParseNumbers(t *testing.T) {
 	}
 	parser := participle.MustBuild[grammar]()
 	_, err := parser.ParseString("", `300 0 x`)
-	require.EqualError(t, err, `grammar.Int: invalid integer "300": strconv.ParseInt: parsing "300": value out of range`)
```

Comment:
@alecthomas I included the previous commit basically just to make it visible what the error message looked like before. IMHO none of the context being added was actually useful and just duplicated the stuff deeper in the error.

The main reason why I removed it had to do with generated parser performance - my parser avoids allocations for errors as much as possible as most errors are later suppressed, so it just collects some error information while parsing and then only allocates the final error once parsing is done. This made that approach tricky as I would need to depend on `fmt` just because of these int/uint/float errors and allocate extra stuff that might later be thrown away. Hope it makes sense.

Reply:
Yep, makes sense 👍